### PR TITLE
Add --version to base command

### PR DESCRIPTION
**Pull request by mhahn**

### CODE REVIEW

**Comment by mhahn** [comment]
output:
```
$ stacker --version
stacker 0.5.0
```

File: stacker/commands/stacker/base.py [hunk 1]
```diff
@@ -132,9 +133,6 @@ def get_context_kwargs(self, options, **kwargs):
         return {}
 
     def add_arguments(self, parser):
-        if not self.subcommands:
-            parser.set_defaults(run=self.run)
```
Comment (phobologic) [inline stacker/commands/stacker/base.py]: This doesn't break the subcommands?

**Comment by phobologic** [comment]
`0.5.0`!? We already have `0.5.1` out!  What're you doing!? ;)

Looks good.

File: stacker/commands/stacker/base.py (hunk 1 above)
Comment (mhahn) [inline stacker/commands/stacker/base.py]: no, we had stopped calling `add_arguments` on the base class, so this code was never getting called.

**Pull request merged**